app/controllers/instructeurs/dossiers_controller.rb

```diff
@@ -72,7 +72,8 @@ def personnes_impliquees
     end
 
     def send_to_instructeurs
-      recipients = Instructeur.find(JSON.parse(params[:recipients]))
+      recipients = params['recipients'].presence || [].to_json
+      recipients = Instructeur.find(JSON.parse(recipients))
```

Comment:
```suggestion
      recipients = params['recipients'].present? ? JSON.parse(params['recipients']) : []
      recipients = Instructeur.find(recipients)
```
mais je suis toujours pas satisfait, j'ai l'impression que y'a un truc évident, du genre `params['recipients'].map(JSON.parse).or([])` mais je le trouve pas.